src/pages/docs/ai-transport/features/sessions-identity/identifying-users-and-agents.mdx

```diff
@@ -0,0 +1,449 @@
+await announcementsChannel.subscribe((msg) => console.log(msg)); // succeeds
+```
+</Code>
+
+<Aside data-type="note">
+Ably recommends using token authentication with short-lived tokens and minimal capabilities required for each user to limit the window of exposure if a token is compromised. For more information, see [When to use token auth](/docs/auth/token#when).
+</Aside>
+
+### Agent capabilities <a id="agent-capabilities"/>
+
+When using API key authentication, provision API keys through the [Ably dashboard](https://ably.com/dashboard) or [Control API](/docs/account/control-api) with only the capabilities required by the agent.
+
+The following example uses the Control API to create an API key with specific capabilities for a weather agent:
+
+<Aside data-type="note">
+To use the Control API, you must first [create an access token](/docs/platform/account/control-api#creating-access-token) in the Ably dashboard. You'll also need your [app ID](/docs/platform/account/control-api#app-id).
+</Aside>
+
+<Code>
+```shell
+curl --location --request POST 'https://control.ably.net/v1/apps/{{APP_ID}}/keys' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer ${ACCESS_TOKEN}' \
+--data-raw '{
+    "name": "weather-agent-key",
+    "capability": {
+        "org:acme:weather:*": ["publish", "subscribe"]
+    }
+}'
+```
+</Code>
+
+This creates an API key that can only publish and subscribe on channels matching `org:acme:weather:*`. The agent can then use this key to authenticate:
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from "ably";
+
+// Use an API key that was provisioned with capabilities to
+// publish and subscribe on channels matching "org:acme:weather:*"
+const ably = new Ably.Realtime({
+  key: "{{API_KEY}}"
+});
+
+const weatherChannel = ably.channels.get("org:acme:weather:{{RANDOM_CHANNEL_NAME}}");
+await weatherChannel.subscribe((msg) => console.log(msg)); // succeeds
+await weatherChannel.publish("update", "It's raining in London"); // succeeds
+
+const otherChannel = ably.channels.get("org:acme:other:{{RANDOM_CHANNEL_NAME}}");
+await otherChannel.subscribe((msg) => console.log(msg)); // fails
+await otherChannel.publish("update", "It's raining in London"); // fails
+```
+</Code>
+
+<Aside data-type="important">
+Following the principle of least privilege, provision API keys with the minimal capabilities needed by each agent. This limits the potential impact if an API key is compromised.
+</Aside>
+
+## Establishing verified identity <a id="identity"/>
+
+Use the [`clientId`](/docs/messages#properties) to identify the user or agent that published a message. For users, use [identified clients](/docs/auth/identified-clients) to assign a trusted `clientId` by embedding the identity in the signed JWT. For trusted agent code, specify the `clientId` directly when instantiating the client instance.
+
+### User identity <a id="user-identity"/>
```

Comment:
The header is User identity, but every where we call it client identity. Do you think we should go with one or the other, or stay as we are?